src/app/components/split-view/index.html

```diff
@@ -0,0 +1,239 @@
+          </sky-docs-visual-example-image>
+      </sky-docs-visual-example>
+    </sky-demo-page-content>
+    <h3>Responsiveness</h3>
+        <sky-docs-visual-example>
+          <sky-docs-visual-example-text>
+            <p>In smaller viewports, the split view automatically switches from a side-by-side view to a panel-switching interaction where users switch between the list and the workspace. The responsive workspace header should include iterator buttons for users to move between items without going back to the list.</p>
+          </sky-docs-visual-example-text>
+          <sky-docs-visual-example-image>
+             <img src="~/assets/img/guidelines/split-view/behavior-responsive.png" />
+          </sky-docs-visual-example-image>
+      </sky-docs-visual-example>
+
+    <sky-demo-page-content sectionHeading="Layout">
+      <sky-docs-visual-example>
+        <sky-docs-visual-example-text>
+          <p>A split view should be the last item on a page. Do not place additional content below it.</p>
+        </sky-docs-visual-example-text>
+        <sky-docs-visual-example-image>
+          <img src="~/assets/img/guidelines/split-view/layout-last-item.png" />
+          <div class="caption caption-danger sky-text-danger">Don't place content below the split view.</div>
+        </sky-docs-visual-example-image>
+      </sky-docs-visual-example>
+    </sky-demo-page-content>
+
+    <sky-demo-page-content sectionHeading="Related information">
+      <h3>Components</h3>
+      <ul>
+        <li><a routerLink="/components/repeater">Repeater</a></li>
+        <li><a routerLink="/components/summary-actionbar">Summary action bar</a></li>
+      </ul>
+    </sky-demo-page-content>
+
+  </sky-tab>
+
+  <sky-tab
+    tabHeading="Development">
+
+  <sky-demo-page-module-info
+    skyuxModule="SkySplitViewModule"
+    packageName="@skyux/split-view"
+    githubRepo="skyux-split-view"
+  >
+  </sky-demo-page-module-info>
+
+  <sky-demo-page-properties sectionHeading="Split view properties">
+    <sky-demo-page-property
+      propertyName="backButtonText"
+      isOptional="true"
+      defaultValue="Back to list"
+    >
+      Specifies a label for the button that appears in the workspace header in responsive mode. This button returns users to the list. This property accepts <sky-code>string</sky-code> values.
+    </sky-demo-page-property>
+    <sky-demo-page-property
+      propertyName="messageStream"
+      isOptional="true"
+    >
+      Provides an observable to send commands to the split view component. The commands should respect <a stacheRouterLink="." fragment="skysplitviewmessage-type">the <sky-code>SkySplitViewMessage</sky-code> type</a>.
+    </sky-demo-page-property>
+  </sky-demo-page-properties>
+
+  <sky-demo-page-properties sectionHeading="SkySplitViewMessage type">
+    <sky-demo-page-property
+      propertyName="type"
+      isOptional="true"
+    >
+      Sets <sky-code>SkySplitViewMessage</sky-code> to one of the following types:
```

Comment:
Are there plans to add more types?

Reply:
Unsure at this point. We might have types for iterators down the road. Is there a better way to word this?

Reply:
Given that there is only 1 item and we don't have current plans to add more, I wonder if it makes sense to change this to "Sets the type for `SkySplitViewMessage`. Currently, <sky-code>SkySplitViewMessageType.FocusWorkspace</sky-code>, which places focus on the first focusable element in the workspace, is the only available type." What do you think of that?

Reply:
Yep that's fine with me!

Reply:
Done.